openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -472,6 +467,21 @@ def _delete_cookie(request, response):
         httponly=settings.SESSION_COOKIE_HTTPONLY or None,
     )
 
+    # Log the cookie, but cap the length and base64 encode to make sure nothing
+    # malicious gets directly dumped into the log.
+    cookie_header = request.META.get('HTTP_COOKIE', '')[:4096]
+    log.warning(
+        u"Malformed Cookie Header? First 4K, in Base64: %s",
+        b64encode(cookie_header)
+    )
+
+    # Note, there is no request.user attribute at this point.
+    if hasattr(request, 'session') and hasattr(request.session, 'session_key'):
+        log.warning(
+            u"SafeCookieData deleted session cookie for session %s",
+            request.session.session_key
+        )
```

Comment:
To be extra paranoid, do you want to encapsulate these changes in a `try-except` block?

Reply:
I think it's okay as is, but if we want to go extra paranoid, I think maybe in a followup PR -- just because the tests take a while to run, and I'd rather get this in ASAP to not hold up the release.